### DISCUSSION TITLE
Sumatra toolbar buttons are too small

### DISCUSSION THREAD

**Discussion by ltcomdata** [sumatrapdfreader/sumatrapdf#4181] [General]
With the recent changes to Sumatra to be more friendly to higher-resolution screens the toolbar buttons are now simply too small.  I understand that some applications benefit from taking advantage of increased resolution.  But why must the GUI elements shrink as a result?  Could we have an option to make the toolbar icons bigger?

**Comment by GitHubRulesOK** [comment]
the option is there in advanced settings
ToolbarSize = 100
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/cccb6d5b-c7ab-417d-bdd8-675a28f97320)

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/5a82ad5a-6fd4-4444-9990-44102ae497ea)

**Reply by ltcomdata** [reply]
Thanks!

**Comment by kjk** [comment]
Also, in pre-release you can set `UIFontSize = 16` (or whatever) advanced setting and toolbar / menu / tabs / dialogs will use this font.
